mods/ctf_pvp_engine/ctf_flag/flag_func.lua

```diff
@@ -210,6 +210,14 @@ ctf_flag = {
 					minetest.chat_send_player(name, "You are at peace with this team!")
 					return
 				end
+				
+				local glitcher = players_glitching[name]
+				if glitcher then
+					local player = minetest.get_player_by_name(name)
```

Comment:
`player` is only used in the next line after which it's scope expires. You can directly call `set_pos` like so:

`minetest.get_player_by_name(name):set_pos(pos)`